setup/web_server_configuration.rst

```diff
@@ -380,7 +185,43 @@ The **minimum configuration** to get your application running under Nginx is:
 
 For advanced Nginx configuration options, read the official `Nginx documentation`_.
 
+Caddy
+-----
+
+When using Caddy on the server, you can use a configuration like this:
+
+.. code-block:: raw
+
+    # /etc/caddy/Caddyfile
+    domain.tld, www.domain.tld {
+        root * /var/www/project/public
+
+        # serve files directly if they can be found (e.g. CSS or JS files in public/)
+        encode zstd gzip
+        file_server
+
+
+        # otherwise, use PHP-FPM (replace "unix//var/..." with "127.0.0.1:9000" when using TCP)
+        php_fastcgi unix//var/run/php/php7.4-fpm.sock {
+            # optionally set the value of the environment variables used in the application
+            # env APP_ENV "prod"
+            # env APP_SECRET "<app-secret-id>"
+            # env DATABASE_URL "mysql://db_user:db_pass@host:3306/db_name"
```

Comment:
I would suggest to add the `resolve_root_symlink` directive to this part: it helps in case deployments where the codebase is symlinked.
In case of symlinks the OpCache uses the "symlinked path" to identify the cached files, thus caches are not refreshed when a new deployment is done and the symlink changed to point to the new files.

More details: https://caddy.community/t/root-symlink-folder-updates-and-caddy-reload-not-working/10557

Reply:
IMHO we should keep the example as minimal as possible. This kind of edge cases should be documented in the Caddy docs, not here.

Reply:
This is a good suggestion imho, we also document symlink usage in the Apache and Nginx config examples.

It's quite common for deployments in Symfony to use symlinks (due to the popularity of tools like DeployerPHP and Capistrano). So we can save users a couple hours of digging into docs and GitHub issues if we add a small suggestion like this :)

Reply:
This would be my suggestion:
```
        php_fastcgi unix//var/run/php/php8.2-fpm.sock {
            # optionally set the value of the environment variables used in the application
            # env APP_ENV "prod"
            # env APP_SECRET "<app-secret-id>"
            # env DATABASE_URL "mysql://db_user:db_pass@host:3306/db_name"

            # Configure the FastCGI to resolve any symlinks in the root path
            # this ensures that OpCache is suing the destination filenames, instead of the symlinks pointing to the files
            # see https://caddy.community/t/root-symlink-folder-updates-and-caddy-reload-not-working/10557
            resolve_root_symlink
        }
```

Reply:
I agree to keep the resolve symlink in our case deployer is very common and fighting this kind of issues can be avoided if we already add it to the docs :)